Code/GraphMol/DistGeomHelpers/Embedder.cpp

```diff
@@ -270,19 +284,150 @@ bool _boundsFulfilled(const std::vector<int> &atoms,
   return true;
 }
 
-// the minimization using experimental torsion angle preferences
-bool _minimizeWithExpTorsions(
-    RDGeom::PointPtrVect &positions, DistGeom::BoundsMatPtr mmat,
-    double optimizerForceTol, double basinThresh,
-    const std::vector<std::pair<int, int>> &bonds,
-    const std::vector<std::vector<int>> &angles,
-    const std::vector<std::vector<int>> &expTorsionAtoms,
-    const std::vector<std::pair<std::vector<int>, std::vector<double>>>
-        &expTorsionAngles,
-    const std::vector<std::vector<int>> &improperAtoms,
-    const std::vector<int> &atomNums, bool useBasicKnowledge) {
-  RDUNUSED_PARAM(basinThresh);
+namespace EmbeddingOps {
+bool generateInitialCoords(RDGeom::PointPtrVect *positions,
+                           const detail::EmbedArgs &eargs,
+                           const EmbedParameters &embedParams,
+                           RDNumeric::DoubleSymmMatrix &distMat,
+                           RDKit::double_source_type *rng) {
+  bool gotCoords = false;
+  if (!embedParams.useRandomCoords) {
+    double largestDistance =
+        DistGeom::pickRandomDistMat(*eargs.mmat, distMat, *rng);
+    RDUNUSED_PARAM(largestDistance);
+    gotCoords = DistGeom::computeInitialCoords(distMat, *positions, *rng,
+                                               embedParams.randNegEig,
+                                               embedParams.numZeroFail);
+  } else {
+    double boxSize;
+    if (embedParams.boxSizeMult > 0) {
+      boxSize = 5. * embedParams.boxSizeMult;
```

Comment:
5. and -1 are fairly magic here, perhaps these could be named so a user like myself could determine their intent?

Reply:
boxSizeMult is documented here: https://github.com/rdkit/rdkit/blob/master/Code/GraphMol/DistGeomHelpers/Embedder.h#L45

And the 5. is one of those empirical things.